tests/integration-tests/tests/schedulers/test_custom_munge_key.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+from time import sleep
+
+import pytest
+from assertpy import assert_that
+from constants import ENCODE_CUSTOM_MUNGE_KEY
+from remote_command_executor import RemoteCommandExecutor
+from utils import wait_for_computefleet_changed
+
+
+@pytest.mark.usefixtures("instance", "os")
+@pytest.mark.parametrize("use_login_node", [True, False])
+def test_custom_munge_key(
+    region,
+    pcluster_config_reader,
+    clusters_factory,
+    scheduler_commands_factory,
+    store_secret_in_secret_manager,
+    use_login_node,
+):
+    """
+    Test custom munge key config, rotate, update and remove.
+
+    This test contains two case: with LoginNodes and without LoginNodes section.
+    """
```

Comment:
Please expand this _documentation_ for the test with all the main phases of the test:
- generic functionality of the custom munge key on both head node and login nodes;
- check munge key update script behavior after stopping compute fleet only;
- check munge key update script behavior after stopping also the login nodes;
- test updating the cluster removing the munge key;

Reply:
Done!